src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -34,7 +43,7 @@ public static function check_for_updates() {
 	 *
 	 * @return array The debug data for the site.
 	 */
-	public static function debug_data() {
+	public static function debug_data(): array {
```

Comment:
```suggestion
	public static function debug_data() {
```
Let's keep the changes strictly scoped to the purpose of the ticket, which is to abstract `debug_data()` logic into new methods.